packages/vda-sbt-client/test/blockchain-api.test.ts

```diff
@@ -19,11 +33,90 @@ const createBlockchainAPI = (did: any) => {
     })
 }
 
+const claimSBT = async (
+    didAddress: string,
+    sbtType: string,
+    uniqueId: string,
+    sbtURI: string,
+    receiverAddress: string,
+    signedProof: string,
+    keyring: Keyring,
+    blockchainApi: VeridaSBTClient
+) => {
+    const signedDataMsg = ethers.utils.solidityPack(
+        ['string','address'],
+        [`${sbtType}-${uniqueId}-`, didAddress.toLowerCase()]
+    )
+    const signedData = await keyring.sign(signedDataMsg)
+
+    await blockchainApi.claimSBT(
+        sbtType,
+        uniqueId,
+        sbtURI,
+        receiverAddress.toLowerCase(),
+        signedData,
+        signedProof
+    )
+}
+
+const createTestDataIfNotExist = async (
```

Comment:
This function mint & burn SBTs for tests. This can be run only once after the contract deployed.